packages/common/types/ContainerTypes.ts

```diff
@@ -106,7 +107,7 @@ export interface GradeReport {
     // significantly impact the performance of the dashboard.
     // Use attachments instead for large bits of data you wish
     // to persist.
-    custom: {};
+    custom: any;
```

Comment:
@braxtonhall Did the object type have limitations? This totally looks fine, but just curious.

Reply:
If you use `{}` it expects _just_ an empty object.
I think it would probably be more correct to do something like
```typescript
custom: {[key: string]: any};
```

Reply:
Roger that. The typing issue makes a lot of sense. Thanks!